src/test/end-to-end/paths/caseworker/grantOfProbatePath.js

```diff
@@ -190,7 +190,7 @@ Scenario('01 BO Grant of Representation E2E - Grant issued', async function (I)
     //
     // This is as far as we can currently get locally due to bulk printing issue
     await I.enterEventSummary(caseRef, nextStepName);
-    // 
+    //
```

Comment:
Adding extra spaces could lead to failures on e2e , does it work fine in local ?

Reply:
yes works fine on local, this was yarn eslint-test --fix which did this automatically.
Previous merges to master must have omitted this de-linting step